packages/flutter/lib/src/material/search.dart

```diff
@@ -193,12 +192,7 @@ abstract class SearchDelegate<T> {
     assert(context != null);
     final ThemeData theme = Theme.of(context);
     assert(theme != null);
-    return theme.copyWith(
-      primaryColor: Colors.white,
```

Comment:
is there a way to keep the copyWith here but use the correct colors depending on light/dark theme?